components/support/nimbus-fml/src/workflows.rs

```diff
@@ -158,6 +161,63 @@ pub(crate) fn fetch_file(files: &LoaderConfig, nm: &str) -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn validate(cmd: &ValidateCmd) -> Result<()> {
+    let files: FileLoader = TryFrom::try_from(&cmd.loader)?;
+    let cwd = cmd.loader.cwd.to_str().unwrap();
+
+    let filename = &cmd.manifest;
+    let file_path = files.file_path(filename)?;
+    let parser: Parser = Parser::new(files, file_path.clone())?;
+    let mut loading = HashSet::new();
+    let manifest_front_end = parser.load_manifest(&file_path, &mut loading)?;
+
+    println!(
+        "Loaded modules:\n- {}\n",
+        loading
+            .iter()
+            .map(|id| id.to_string().replace(cwd, ""))
```

Comment:
We should probably be a bit smarter about this, and use [a crate](https://crates.io/crates/relative-path).

Currently this outputs a file with a leading slash, suggesting a root path.

Reply:
This might want a method in `ModuleId`.

Reply:
Unless I'm entirely misunderstanding the relative-path crate, I don't think it does what we'd need here. 

By the time a local file name has become a `ModuleId`, it's already been canonicalized, and `RelativePath` doesn't seem to have the ability to convert a canonical path into a relative one.

Is there something I'm missing?